modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -64,6 +64,13 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		AssetListEntryCreateDateComparator.getInstance(false);
 		AssetTagCountComparator.getInstance(false);
 
+		CommercePricingClassCreateDateComparator.getInstance(false);
+        CommerceTierPriceEntryMinQuantityComparator.getInstance(false);
```

Comment:
@Valmir-unicap several wrong spacing in these lines, we need to use tabs not spaces.